activerecord-session_store.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.version     = ActiveRecord::SessionStore::VERSION
   s.summary     = 'An Action Dispatch session store backed by an Active Record class.'
 
-  s.required_ruby_version = '>= 1.9.3'
```

Comment:
Ahh, looking at the failures in CI now I understand what needs to be done. Will fix.